bundles/org.openhab.binding.http/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -112,9 +126,20 @@
 				<default>false</default>
 				<advanced>true</advanced>
 			</parameter>
+			<parameter name="userAgent" type="text">
+				<label>User Agent</label>
+				<description>Sets a custom user agent (default is "Jetty/version", e.g. "Jetty/9.4.20.v20190813").</description>
+				<advanced>true</advanced>
+			</parameter>
 		</config-description>
 	</thing-type>
 
+	<channel-type id="requestDateTime">
+		<item-type>DateTime</item-type>
+		<label>Dummy </label>
```

Comment:
```suggestion
		<label>Dummy</label>
```

I would also suggest to change to a generic label other than "Dummy" since I think it could end up as the actual label in some cases - see openhab/openhab-core#3569.